tests/e2e/gc_block_volume_expansion.go

```diff
@@ -2424,27 +2425,32 @@ func waitForPVCToReachFileSystemResizePendingCondition(client clientset.Interfac
 // checkPvcHasGivenStatusCondition checks if the status condition in PVC
 // matches with the one we want.
 func checkPvcHasGivenStatusCondition(client clientset.Interface, namespace string, pvcName string,
```

Comment:
don't change a check function to wait function, add a new wait function if required
if you see the code change in [vsphere_shared_datastore.go](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1959/files#diff-ad660e62f8bbde64f3eedd7b41195f41e67910008a8836eee2ba5188e49e373d) it is breaking the test logic

Reply:
Added new functionality for wait condition